src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagGeoJsonProcessor.java

```diff
@@ -111,7 +113,7 @@ public void process(final CheckFlagEvent event)
 
     @Override
     @Subscribe
-    public void process(final ShutdownEvent event)
+    public void process(final org.openstreetmap.atlas.event.ShutdownEvent event)
```

Comment:
Why are these changes?

Reply:
Changing to use the classes in atlas, instead of replicating them in atlas checks.